src/code/code_800EC960.c

```diff
@@ -3998,8 +3998,8 @@ void Audio_SetSoundProperties(u8 bankId, u8 entryIdx, u8 channelIdx) {
             panSigned = Audio_ComputeSoundPanSigned(*entry->posX, *entry->posZ, entry->token);
             freqScale = Audio_ComputeSoundFreqScale(bankId, entryIdx) * *entry->freqScale;
             if (D_80130604 == 2) {
-                behindScreenZ = sBehindScreenZ[(entry->sfxParams & 0x400) >> 10];
-                if (!(entry->sfxParams & 0x800)) {
+                behindScreenZ = sBehindScreenZ[(entry->sfxParams & SFX_FLAG_10) >> SFX_FLAG_10_SHIFT];
```

Comment:
Just an idea but maybe this bit shouldn't be considered a flag at all since it's used as an index here (and could be extended to multiple values in theory). I don't know if this would be an improvement overall tbh, but it may be something to consider

Reply:
I changed `SFX_FLAG_10` -> `SFX_PARAM_10